docs/rules.md

```diff
@@ -36,3 +36,4 @@
 - Twig
 - DirectoryNameRule
 - FileNameRule
+- FileExtensionRule
```

Comment:
You need to add the extension in src/Standard/Symfony.php.

Reply:
Do you have here some automation system / CI to check it on a Symfony app/demo ?

Or i test manually on a new webapp just to be sure Symfony... follows Symfony conventions ? 👼

Reply:
I dont have any automation yet for Symfony standard no

Reply:
Good for symfony (core repo)